storage/src/adb/current/ordered.rs

```diff
@@ -303,14 +303,8 @@ impl<
         self.last_commit_loc = Some(Location::new_unchecked(self.status.len()));
         self.status.push(true); // Always treat most recent commit op as active.
 
-        // Sync the log and merkleize the MMR updates in parallel.
-        let mmr_fut = async {
-            self.any.mmr.merkleize(&mut self.any.hasher);
-            Ok::<(), Error>(())
-        };
-        try_join!(self.any.log.sync().map_err(Error::Journal), mmr_fut)?;
-
-        Ok(())
+        // Durably persist the log.
+        Ok(self.any.log.sync().await?)
```

Comment:
sync -> sync_data?

Reply:
Since this is a fixed journal I guess it doesn't matter.

Reply:
OK scratch that. I thought we might be adding a commit (aka "sync data") to the Contiguous trait but I see there's no such thing.  (We might want to add it to further unify variable/fixed adbs).